docs/spec/operators/index.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Operators'
+sidebar_position: 0
+description: Documentation for Operators
+---
+
+import DocCardList from '@theme/DocCardList';
+
+Operators are the cornerstones of the Inspektor Gadget architecture. They are
+modular components implementing the same interfaces
+([1](https://pkg.go.dev/github.com/inspektor-gadget/inspektor-gadget@v0.31.0/pkg/operators#DataOperator),
+[2](https://pkg.go.dev/github.com/inspektor-gadget/inspektor-gadget@v0.31.0/pkg/operators#ImageOperator)).
```

Comment:
@mauriciovasquezbernal why to point to a specific version and not to point to the latest:

https://pkg.go.dev/github.com/inspektor-gadget/inspektor-gadget/pkg/operators#DataOperator
https://pkg.go.dev/github.com/inspektor-gadget/inspektor-gadget/pkg/operators#ImageOperator

?

Reply:
I didn't find a way to point to latest, but you're right, latest it better.

Reply:
https://github.com/inspektor-gadget/inspektor-gadget/pull/3440